### PR TITLE
chore: render app on local host with an sdk shim

### PR DESCRIPTION
**Pull request by luqven**
## Description
This PR shims the Contentful SDK when running on a development ENV so that you can render the React application for local development on localhost.

## Before
All testing had to run on contentful itself

## After
The app can run on localhost and be tested on cypress

## Steps to test
- checkout the branch, run `npm install`
- set the `REACT_APP_CTFL_API_KEY` environment variable to be your API key
- run `npm run start`
- run `npm run cypress` OR, if not testing just go open `http://localhost:3000`
- click on e2e tests
- click to run tests

## 🖼️ Screenshots

https://user-images.githubusercontent.com/16711614/200433915-f09dd3c8-09d8-4315-9e42-854c6a6c53ca.mov

## Checklist

- [ ] Read the [contributing guidelines](CONTRIBUTING.md).
- [ ] Each commit follows the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) spec format.
- [ ] Update the readme (if applicable).
- [ ] Update or add any necessary API documentation (if applicable)
- [ ] All existing unit tests are still passing (if applicable).

- [ ] Add some [steps](#steps-to-test) so we can test your bug fix or feature (if applicable).
- [ ] Add new passing unit tests to cover the code introduced by your PR (if applicable).
- [ ] Any breaking changes are specified on the commit on which they are introduced with `BREAKING CHANGE` in the body of the commit.
- [ ] If this is a big feature with breaking changes, consider opening an issue to discuss first. This is completely up to you, but please keep in mind that your PR might not be accepted.

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #140** <a href="https://app.graphite.dev/github/pr/imgix/contentful/140" target="_blank">Graphite</a> 
    * **PR #139** <a href="https://app.graphite.dev/github/pr/imgix/contentful/139" target="_blank">Graphite</a> 
      * **PR #141** <a href="https://app.graphite.dev/github/pr/imgix/contentful/141" target="_blank">Graphite</a> 
        * **PR #142** <a href="https://app.graphite.dev/github/pr/imgix/contentful/142" target="_blank">Graphite</a> 
          * **PR #143** <a href="https://app.graphite.dev/github/pr/imgix/contentful/143" target="_blank">Graphite</a> 
            * **PR #144** <a href="https://app.graphite.dev/github/pr/imgix/contentful/144" target="_blank">Graphite</a> 
              * **PR #145** <a href="https://app.graphite.dev/github/pr/imgix/contentful/145" target="_blank">Graphite</a> 
                * **PR #146** <a href="https://app.graphite.dev/github/pr/imgix/contentful/146" target="_blank">Graphite</a> 
                  * **PR #147** <a href="https://app.graphite.dev/github/pr/imgix/contentful/147" target="_blank">Graphite</a> 
                    * **PR #148** <a href="https://app.graphite.dev/github/pr/imgix/contentful/148" target="_blank">Graphite</a> 
                      * **PR #149** <a href="https://app.graphite.dev/github/pr/imgix/contentful/149" target="_blank">Graphite</a> 
                        * **PR #150** <a href="https://app.graphite.dev/github/pr/imgix/contentful/150" target="_blank">Graphite</a> 
                          * **PR #153** <a href="https://app.graphite.dev/github/pr/imgix/contentful/153" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/contentful/153?utm_source=stack-comment).

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #140** <a href="https://app.graphite.dev/github/pr/imgix/contentful/140" target="_blank">Graphite</a> 
    * **PR #139** <a href="https://app.graphite.dev/github/pr/imgix/contentful/139" target="_blank">Graphite</a> 
      * **PR #141** <a href="https://app.graphite.dev/github/pr/imgix/contentful/141" target="_blank">Graphite</a> 
        * **PR #142** <a href="https://app.graphite.dev/github/pr/imgix/contentful/142" target="_blank">Graphite</a> 
          * **PR #143** <a href="https://app.graphite.dev/github/pr/imgix/contentful/143" target="_blank">Graphite</a> 
            * **PR #144** <a href="https://app.graphite.dev/github/pr/imgix/contentful/144" target="_blank">Graphite</a> 
              * **PR #145** <a href="https://app.graphite.dev/github/pr/imgix/contentful/145" target="_blank">Graphite</a> 
                * **PR #146** <a href="https://app.graphite.dev/github/pr/imgix/contentful/146" target="_blank">Graphite</a> 
                  * **PR #147** <a href="https://app.graphite.dev/github/pr/imgix/contentful/147" target="_blank">Graphite</a> 
                    * **PR #148** <a href="https://app.graphite.dev/github/pr/imgix/contentful/148" target="_blank">Graphite</a> 
                      * **PR #149** <a href="https://app.graphite.dev/github/pr/imgix/contentful/149" target="_blank">Graphite</a> 
                        * **PR #150** <a href="https://app.graphite.dev/github/pr/imgix/contentful/150" target="_blank">Graphite</a> 
                          * **PR #153** <a href="https://app.graphite.dev/github/pr/imgix/contentful/153" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/contentful/153?utm_source=stack-comment).

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #140** <a href="https://app.graphite.dev/github/pr/imgix/contentful/140" target="_blank">Graphite</a> 
    * **PR #139** <a href="https://app.graphite.dev/github/pr/imgix/contentful/139" target="_blank">Graphite</a> 
      * **PR #141** <a href="https://app.graphite.dev/github/pr/imgix/contentful/141" target="_blank">Graphite</a> 
        * **PR #142** <a href="https://app.graphite.dev/github/pr/imgix/contentful/142" target="_blank">Graphite</a> 
          * **PR #143** <a href="https://app.graphite.dev/github/pr/imgix/contentful/143" target="_blank">Graphite</a> 
            * **PR #144** <a href="https://app.graphite.dev/github/pr/imgix/contentful/144" target="_blank">Graphite</a> 
              * **PR #145** <a href="https://app.graphite.dev/github/pr/imgix/contentful/145" target="_blank">Graphite</a> 
                * **PR #146** <a href="https://app.graphite.dev/github/pr/imgix/contentful/146" target="_blank">Graphite</a> 
                  * **PR #147** <a href="https://app.graphite.dev/github/pr/imgix/contentful/147" target="_blank">Graphite</a> 
                    * **PR #148** <a href="https://app.graphite.dev/github/pr/imgix/contentful/148" target="_blank">Graphite</a> 
                      * **PR #149** <a href="https://app.graphite.dev/github/pr/imgix/contentful/149" target="_blank">Graphite</a> 
                        * **PR #150** <a href="https://app.graphite.dev/github/pr/imgix/contentful/150" target="_blank">Graphite</a> 
                          * **PR #153** <a href="https://app.graphite.dev/github/pr/imgix/contentful/153" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/contentful/153?utm_source=stack-comment).

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #140** <a href="https://app.graphite.dev/github/pr/imgix/contentful/140" target="_blank">Graphite</a> 
    * **PR #139** <a href="https://app.graphite.dev/github/pr/imgix/contentful/139" target="_blank">Graphite</a> 
      * **PR #141** <a href="https://app.graphite.dev/github/pr/imgix/contentful/141" target="_blank">Graphite</a> 
        * **PR #142** <a href="https://app.graphite.dev/github/pr/imgix/contentful/142" target="_blank">Graphite</a> 
          * **PR #143** <a href="https://app.graphite.dev/github/pr/imgix/contentful/143" target="_blank">Graphite</a> 
            * **PR #144** <a href="https://app.graphite.dev/github/pr/imgix/contentful/144" target="_blank">Graphite</a> 
              * **PR #145** <a href="https://app.graphite.dev/github/pr/imgix/contentful/145" target="_blank">Graphite</a> 
                * **PR #146** <a href="https://app.graphite.dev/github/pr/imgix/contentful/146" target="_blank">Graphite</a> 
                  * **PR #147** <a href="https://app.graphite.dev/github/pr/imgix/contentful/147" target="_blank">Graphite</a> 
                    * **PR #148** <a href="https://app.graphite.dev/github/pr/imgix/contentful/148" target="_blank">Graphite</a> 
                      * **PR #149** <a href="https://app.graphite.dev/github/pr/imgix/contentful/149" target="_blank">Graphite</a> 
                        * **PR #150** <a href="https://app.graphite.dev/github/pr/imgix/contentful/150" target="_blank">Graphite</a> 
                          * **PR #153** <a href="https://app.graphite.dev/github/pr/imgix/contentful/153" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/contentful/153?utm_source=stack-comment).

File: src/index.tsx [hunk 1]
```diff
@@ -1,28 +1,59 @@
...
 } from 'contentful-ui-extensions-sdk';
-import '@contentful/forma-36-react-components/dist/styles.css';
-import '@contentful/forma-36-fcss/dist/styles.css';
-import '@contentful/forma-36-tokens/dist/css/index.css';
+import { render } from 'react-dom';
 
 import Config from './components/ConfigScreen';
-import Field from './components/Field/';
 import Dialog from './components/Dialog/';
+import Field from './components/Field/';
```
Comment (sherwinski) [inline src/index.tsx:15]: Do you mind disabling code formatting on save? Each of the diffs on these PRs are getting buffed with import statements being shuffled around 😅

**Review by sherwinski — APPROVED** [review]
Looks good but do we need any of the following files?

`cypress/fixtures/example.json`
`cypress/support/commands.ts`
`cypress/support/e2e.ts`

**Comment by luqven** [comment]
@sherwinski merging his in but tagging you one more time just in case you want to see it once more after the force push.

**Pull request merged**